qcodes/tests/dataset/dond/test_do2d.py

```diff
@@ -154,9 +154,12 @@ def test_do2d_output_data(_param, _param_complex, _param_set, _param_set_2):
     )
     data = exp[0]
 
-    assert data.parameters == (
```

Comment:
as agreed offline, let's keep at least some coverage for (the discouraged) `.parameters` to ensure the order is preserved

Reply:
added to test_dataset_basic